.dockerignore

```diff
@@ -1 +0,0 @@
-.gitignore
```

Comment:
Can you explain what prompted this change? In particular removing the .gitignore